src/lesson_04/hometask_04/Task10.java

```diff
@@ -0,0 +1,41 @@
+package lesson_04.hometask_04;
+/*A sequence of natural numbers we will call zigzag if the following conditions are
+        met for its elements:
+        N1 < N2 > N3 < N4 > N5 < … > Ni < Nj > … < Nn
+or
+        N1 > N2 < N3 > N4 < N5 > … < Ni > Nj < … > Nn
+        Write a program that checks whether a sequence of numbers entered into a
+        one-dimensional array meet the above requirements.
+        Example:
+        Input:
+        [1, 6, 3, 7, 2, 9, -2, 12, 5]
+        [1, 2, 3, 4, 5, 6, 7, 8, 9]
+        Output:
+        Zigzag
+        Not a Zigzag*/
+public class Task10 {
+    public static void main(String[] args) {
+        int[] array1 = new int[] {1, 6, 3, 7, 2, 9, -2, 12, 5};
```

Comment:
Could you test with:
 {1, 2, 3, 7, 2, 9};
 It is not a zigZag arr, but your output is different.